extensions/odoo_theme/static/style.scss

```diff
@@ -291,15 +285,6 @@ header.o_main_header {
                 display: inline-block;
                 padding: .4rem 0;
                 color: inherit;
-                opacity: .75;
-
-                &.current, &:hover {
-                    opacity: 1;
-                }
```

Comment:
By removing this, there's no clear indicator of where the user is in the toc.
I recommend at least adding a bolder font-weight on hovered & current `a` elements.